docs/ManualDeploymentGuideScalarDLonAWS.md

```diff
@@ -0,0 +1,231 @@
+# Deploy Scalar DL on AWS
+
+Scalar DL is a database-agnostic distributed ledger middleware containerized with Docker. 
+It can be deployed on various platforms and is recommended to be deployed on managed services for production to achieve high availability and scalability, and maintainability. 
+This guide shows you how to manually deploy Scalar DL on a managed database service and a managed Kubernetes service in Amazon Web Services (AWS) as a starting point of deploying Scalar DL for production.
+
+## What we create
+
+![image](images/network_diagram_eks.png)
+
+In this guide, we will create the following components.
+
+* A VPC with NAT gateway
+* An EKS cluster with two Kubernetes node groups
+* A managed database service (you can choose one of them)
+    * DynamoDB
+    * RDS (MySQL/PostgreSQL)
+* 1 Bastion instance with a public IP
+* Amazon CloudWatch
+
+## Step 1. Configure your network
+
+Configure a secure network with your organizational or application standards. 
+Scalar DL handles highly sensitive data of your application, so you should create a highly secure network for production. This section will help you to configure a secure network for Scalar DL deployments.
+
+### Requirements
+ 
+* You must have a VPC, with NAT gateways on private subnets. NAT gateway is necessary to enable internet access for Kubernetes node group subnets.
+* You must have at least 2 subnets in different AZs for the Kubernetes cluster. This is mandatory to create an EKS cluster.
+* You must have at least one private subnet for the managed database other than Dynamo DB.
+* You must create subnets with the prefix at least _/24_ for the Kubernetes cluster to work without issues even after scaling. 
+
+### Recommendations
+
+* Kubernetes should be private in production and should provide secure access with SSH or VPN. You can use a bastion server as a host machine for secure access to the private network.
+* Subnets should be in multiple availability zones, to enable fault tolerance for the production use.
+* Kubernetes cluster should be public to enable the envoy public endpoint(Scalar DL can be accessed from the internet). The Kubernetes cluster in public is not recommended for production.
```

Comment:
Is there any relationship between `kubernetes cluster in public` and `envoy public endpoint`?

Reply:
Yes. The Kubernetes cluster should have `public subnets` and API endpoint access should be `public`.

Reply:
>and API endpoint access should be public.

Are you sure?
Since the public endpoint of envoy only places `alb` and `nlb` in public subnets, I thought it would be OK if the api endpoint was private. (Need to operate k8s cluster from instances in vpc such as bastion server)

Have you tested it, or is there documentation somewhere?

Reply:
Yes, you are right, API endpoint access can be `private`. I tested and confirmed.

Reply:
So can we modify it?

```suggestion
* Kubernetes cluster should have public subnets to enable the envoy public endpoint(Scalar DL can be accessed from the internet). The Kubernetes cluster in public is not recommended for production.
```

Reply:
Thank you. Document modified.